### DISCUSSION TITLE
wrong buffer detection

### DISCUSSION THREAD

**Discussion by TwelveNight** [LazyVim/LazyVim#2860] [Q&A]
I don`t know why it happen,it seems random
What I do is change the dir from A dir to B dir,but it is not happen always,but when it is happen,it can not be fix.
This make me telescope and neotree can not work correctly
Only when I change the dir name can fix it,but when I change back,it will reappear

https://github.com/LazyVim/LazyVim/assets/113519679/ac53fe5e-54e1-4199-a538-8df05e028b51

I find something like `vim.g.root_spec = { "cwd" }`,but this is not work for me

**Comment by ivanlee1999** [comment]
faced the same problem. Using telescope or neo-tree to open a file, but the buffer of the file seems to have a wrong cwd which is outside of the root dir
